src/PrestaShopBundle/Translation/Locale/Converter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Translation\Locale;
```

Comment:
Should it be in Core instead?

Reply:
I had this thinking too (as for all the new classes I've introduced).

The thing is... isn't harder to look at 2 folders instead to one when looking for translations stuff? It's already planned to move `PrestaShopBundle/Translation` to `Core/Translation` but it should be done in one pull request while guarantying all deprecations and BC safe.

I've ended up not moving nothing in this pull request :angel:

Reply:
But you did but moving methods :trollface:

Reply:
only removing the methods created with the same content of the parent methods :trollface:, looks like some people forgot what inheritance means ahahaha.

Also the convertion of locale shouldn't be managed in the AbstractProvider (SRP etc...), if you want me to move it to `Core` namespace now, I'll do it np :+1:  tell me boss !

Reply:
class Moved to the Core namespace (and fixed according to @eternoendless comments)